src/fundus/publishers/de/br.py

```diff
@@ -14,6 +15,8 @@
 
 class BRParser(ParserProxy):
```

Comment:
`publishing_date` parsing fails for Meldungen, since the publishingDate attribute does not exist in that [case]( https://www.br.de/nachrichten/meldung/messerangreifer-von-moers-war-wohl-psychisch-krank%2C3006aa526). It is mentioned loosely in the source, so perhaps it could be parsed from there.